docker/postgis/postgresql_init_data/jobs_table.sql

```diff
@@ -0,0 +1,11 @@
+create table if not exists jobs (
+	id serial primary key,
+	minx double precision not null,
+	miny double precision not null,
+	maxx double precision not null,
+	maxy double precision not null,
+	transform_to varchar(40),
+	data_id varchar,
```

Comment:
This may be a candidate for type `uuid`.

Reply:
good point. this will be done in a follow up